README.md

```diff
@@ -109,7 +134,7 @@ Write this grammar using EBNF with common extensions
 
 ## A2.3
 
-«replace this with your answer»
+fsm.png
```

Comment:
Your FSM is not valid. 

A state machine has states and transitions that join them. If you are in a state and an event happens, then the state looks for the transition labelled by that event and follows it to the next state.

It looks to me as if you've associated the events with the states themselves. So, for example, when you're in S1, you have two unlabelled transitions to S2 and S3. How is the state machine to know which to take?

I was looking for something like this:

![image](https://user-images.githubusercontent.com/10648/78617608-7df88a80-783d-11ea-9003-7415e6aa84ce.png)

# -4